drivers/disk/Kconfig.sdmmc

```diff
@@ -41,6 +41,13 @@ config SDMMC_USDHC
 	  File system on a SDHC card accessed over
 	  NXP USDHC.
 
+config SDMMC_USDHC_DISABLE_V180
```

Comment:
Although this is to solve the problem of hardware design. But I think this is close to the configuration of the software, because no matter how the external hardware is designed, the SDCard can only work under 3.3V through this configuration item.

Reply:
I think something like this is better to maintain if it is configured via devicetree, I can also be wrong. @MaureenHelm , your opinion?

> Maybe default enable SDMMC_USDHC_DISABLE_V180 item through the devicetree? For example, add a bool attribute of vselect to nxp,imx-usdhc.yaml, Can you give some suggests? Thank you very much!

e.g. Linux Kernel binding for esdhc controller has voltage-ranges property, maybe that is the way to go,
https://www.kernel.org/doc/Documentation/devicetree/bindings/mmc/fsl-esdhc.txt

Reply:
Yes, I agree this should be configured via device tree.

Reply:
@jfischer-no Thanks for your information, I found a more suitable node attribute
https://www.kernel.org/doc/Documentation/devicetree/bindings/mmc/mmc-controller.yaml
```
  no-1-8-v:
    $ref: /schemas/types.yaml#/definitions/flag
    description:
      When specified, denotes that 1.8V card voltage is not supported
      on this system, even if the controller claims it.
```

Reply:
@lgl88911That looks good.